lib/android-tools-info.ts

```diff
@@ -343,45 +337,24 @@ export class AndroidToolsInfo implements IAndroidToolsInfo {
 		return (() => {
 			if (!this.installedTargetsCache) {
 				try {
```

Comment:
I've added the mentioned else statement, but I prefer having the try-catch as well.